### DISCUSSION TITLE
Right panel shows nothing

### DISCUSSION THREAD

**Discussion by cleyjulio** [integrated-application-development/delphilint#44] [Q&A] [answered]
When analyzing any file and clicking on an issue, nothing appears in the right panel. This already happened in the previous version, I removed it, installed the new one, reinstalled it and it still doesn't appear. How can I fix this? Note: I'm running Delphi 12.1 Patch 1 on a Windows 10 VM

![tela](https://github.com/integrated-application-development/delphilint/assets/8240737/57379b1c-f6b4-425c-a864-48eadf3b86d7)

**Comment by zaneduffield** [comment]
Hi @cleyjulio, this might be because the [requirement](https://github.com/integrated-application-development/delphilint?tab=readme-ov-file#installation) for the Edge browser components is not met.
> Microsoft Edge 79.0.309+

Could you check what version of Edge is installed?

**Reply by cleyjulio** [reply]
My installed version is 
Microsoft Edge
Versão 124.0.2478.97 (Compilação oficial) (64 bits)

**Reply by zaneduffield** [reply]
Thanks for checking that. The problem must be elsewhere.

I'll leave this with @fourls, then.

**Comment by fourls** [comment]
Hi @cleyjulio,

Looks like a problem with the embedded browser failing to load the Edge DLL properly. We've had no shortage of problems with this, Delphi certainly doesn't seem to make it easy 😄 

We've been able to reproduce a similar issue locally and we're trying some things to see if they resolve it. Our problems seem to be resolved by running Delphi in administrator mode, but that is obviously not ideal.

Hopefully I'll have some more for you soon.

**Reply by cleyjulio** [reply]
Hi @fourls, 
I run Delphi in administrator mode, but the problem persist.

**Reply by fourls** [reply]
OK, no worries. I might upgrade this to an issue so that we can track it.

**Reply by fourls** [reply]
#45 raised.

**Reply by fourls** [reply]
@cleyjulio, would you be able to check the length of your PATH environment variable? The problem on our machine was that the PATH was more than 2047 characters long, which I guess the Delphi IDE was cutting off.

Clearing some entries from your PATH might fix your problem. I'm writing a longer-term fix now, but in the meantime hopefully that will resolve your issue.

**Reply by cleyjulio** [reply]
This option in library path?

**Reply by fourls** [reply]
PATH in your Windows environment variables. 

You can check the length of your PATH by running `$env:PATH.length` in a PowerShell terminal, and there's instructions [here](https://www.java.com/en/download/help/path.html) for how to access and change your PATH.

**Reply by cleyjulio** [reply]
I check my Windows Path... the length is 1212...
The embedded browser failing again =(
![image](https://github.com/integrated-application-development/delphilint/assets/8240737/b68fd502-867f-4f2e-805c-2078adea1cf3)

**Reply by cirras** [reply]
Hi @cleyjulio,

I've built the latest code for you, which includes the fix from PR #47.
[DelphiLint-1.1.0+dev.197b49d-Athens.zip](https://github.com/user-attachments/files/15741845/DelphiLint-1.1.0%2Bdev.197b49d-Athens.zip)

Can you check and see if that resolves your issue?

**Reply by cleyjulio** [reply]
Hi! Worked fine!! Thanks!

**Reply by cirras** [reply, marked as answer]
This issue is resolved in [DelphiLint v1.1.1](https://github.com/integrated-application-development/delphilint/releases/tag/v1.1.1).

**Comment by KarlheinzJansen** [comment]
I have had the same issue. With the test version it works fine for me.
